pkg/decoders/network-events.go

```diff
@@ -0,0 +1,63 @@
+package decoders
+
+import (
+	"encoding/json"
+
+	"github.com/netobserv/network-observability-console-plugin/pkg/model/fields"
+	ovnmodel "github.com/ovn-org/ovn-kubernetes/go-controller/observability-lib/model"
+	"github.com/sirupsen/logrus"
+)
+
+var dlog = logrus.WithField("module", "decoders")
+
+func NetworkEventsToString(in string) string {
+	line := make(map[string]any)
+	if err := json.Unmarshal([]byte(in), &line); err != nil {
+		dlog.Errorf("Could not decode NetworkEvent: %v", err)
+		return in
+	}
+	if ne, found := line[fields.NetworkEvents]; found {
+		if neList, isList := ne.([]any); isList {
+			var messages []string
+			for _, item := range neList {
+				if neItem, isMap := item.(map[string]any); isMap {
+					messages = append(messages, networkEventItemToString(neItem))
+				}
+			}
+			line[fields.NetworkEvents] = messages
+			b, err := json.Marshal(line)
+			if err != nil {
+				dlog.Errorf("Could not reencode NetworkEvent: %v", err)
+				return in
+			}
+			return string(b)
+		}
+	}
+	return in
+}
+
+func networkEventItemToString(in map[string]any) string {
+	if msg := getAsString(in, "Message"); msg != "" {
+		return msg
+	}
+	if feat := getAsString(in, "Feature"); feat == "acl" {
+		aclObj := ovnmodel.ACLEvent{
+			Action:    getAsString(in, "Action"),
+			Actor:     getAsString(in, "Type"),
+			Name:      getAsString(in, "Name"),
+			Namespace: getAsString(in, "Namespace"),
+			Direction: getAsString(in, "Direction"),
+		}
+		return aclObj.String()
```

Comment:
I don't think its worse add ovn pkg dependency to build the string, u can just build it urself using the same pattern and and avoid additional dependencies

Reply:
That's what I did at first (see https://github.com/netobserv/network-observability-console-plugin/pull/647/commits/089040eedd654756989940ba9a3574fff4e61333#diff-ac6bc2fb6ae2ff5deb13e82b18d432e5473729ee0408fa2baf2860c204fa6c0a ) .. but came back from there to avoid duplicating the logic, and avoid having ovn logic here. Also when more events are added later, they'll land here more easily with just a lib upgrade.